Symfony/CS/Tests/Fixer/Symfony/PhpdocInlineTagFixerTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * Test PhpdocInlineTagFixer.
+ */
+class PhpdocInlineTagFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideTestFixInlineDocCases
+     */
+    public function testFixInlineDoc($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideTestFixInlineDocCases()
+    {
+        $cases = array(
+            array(
+                '<?php
+    /**
+     * { test }
+     * {@inheritdoc test}
+     * test other comment
+     * {@inheritdoc test} a
+     * {@inheritdoc test} b
+     * {@inheritdoc test} c
+     * {@inheritdoc foo bar.} d
+     * {@inheritdoc foo bar.} e
+     * {@inheritdoc test} f
+     * end comment {@inheritdoc here we are done} @spacepossum {1}
+     */
+',
+                '<?php
+    /**
+     * { test }
```

Comment:
Thanks for looking into this :+1: 
A small test shows that you are correct;
```
{link} { link } @{link} {@link} @ {link} {{@link}} @{{link}}
```
is changed to:
```
{@link} {@link } {@link} {@link} @ {@link} {@link} {@link}
```
That leaves the question, do we want this, or should I down scope to `inheritdoc` only?

(and shows that the current regex doesn't remove the right space between the `link` and `}` :/ )